packages/service-client/odsp-client/src/interfaces.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IMember, IServiceAudience } from "@fluidframework/fluid-static";
+import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+import { ITokenProvider } from "@fluidframework/azure-client";
+import { type IConfigProviderBase } from "@fluidframework/telemetry-utils";
+import { type IUser } from "@fluidframework/protocol-definitions";
+
+/**
+ * OdspConnectionConfig defines the necessary properties that will be applied to all containers
+ * created by an OdspClient instance. This includes callbacks for the authentication tokens
+ * required for ODSP.
+ *
+ * @alpha
+ */
+export interface OdspConnectionConfig {
+	/**
+	 * Site url representing ODSP resource location
```

Comment:
Are there more concrete docs we can point to here so users know how to provide this?